gateway/src/main/scala/com/improving/app/gateway/infrastructure/routes/EventGatewayRoutes.scala

```diff
@@ -54,19 +61,80 @@ trait EventGatewayRoutes extends ErrorAccumulatingCirceSupport with StrictLoggin
             onSuccess(
               handler.getAllIds
             ) { allIds =>
-              complete(JsonFormat.toJsonString(allIds))
+              complete(allIds.print)
+            }
+          }
+        } ~ pathPrefix("allData") {
+          pathPrefix("status") {
+            pathPrefix(Segment) { status =>
+              pathPrefix("forOrg") {
+                pathPrefix(Segment) { orgId =>
+                  get {
+                    onSuccess(
+                      handler.getAllIds.map(_.allEventIds.map(id => handler.getEventData(id.id)))
```

Comment:
This code should probably find a method to live in somewhere rather than being inlined in the routes